devices/Sim7080/SIM7080.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+	<metadata>
+		<id>nanoFramework.Iot.Device.SIM7080</id>
+		<version>$version$</version>
+		<title>nanoFramework.Iot.Device.SIM7080</title>
+		<authors>nanoframework</authors>
+		<requireLicenseAcceptance>false</requireLicenseAcceptance>
+		<license type="file">LICENSE.md</license>
+		<releaseNotes>
+		</releaseNotes>
+		<readme>docs\README.md</readme>
+		<developmentDependency>false</developmentDependency>
+		<projectUrl>https://github.com/nanoframework/nanoFramework.IoT.Device</projectUrl>
+		<icon>images\nf-logo.png</icon>
+		<repository type="git" url="https://github.com/nanoframework/nanoFramework.IoT.Device" commit="$commit$"/>
+		<copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+		<description>This package includes the .NET IoT Core binding Iot.Device.SIM7080 for .NET nanoFramework C# projects.</description>
+		<summary>Iot.Device.SIM7080 assembly for .NET nanoFramework C# projects</summary>
+		<tags>nanoFramework C# csharp netmf netnf Iot.Device.SIM7080</tags>
+		<dependencies>
+			<dependency id="nanoFramework.CoreLibrary" version="1.14.2"/>
+			<dependency id="nanoFramework.Runtime.Events" version="1.11.6"/>
+			<dependency id="nanoFramework.System.Collections" version="1.5.18"/>
+			<dependency id="nanoFramework.System.Text" version="1.2.37"/>
+			<dependency id="nanoFramework.System.IO.Ports" version="1.1.60"/>
+			<dependency id="nanoFramework.System.IO.Streams" version="1.1.38"/>
+			<dependency id="nanoFramework.System.Text.RegularExpressions" version="1.1.37"/>
+		</dependencies>
+	</metadata>
+	<files>
+		<file src="bin\Release\Iot.Device.SIM7080.dll" target="lib\Iot.Device.SIM7080.dll"/>
```

Comment:
the file won't exist on non Windows system because of the case. Please adjust with the name which is in nfproj so "Sim7080".

Reply:
Changed!